packages/cli/__tests__/zosjobs/__unit__/delete/old-jobs/__snapshots__/OldJobs.handler.unit.test.ts.snap

```diff
@@ -0,0 +1,253 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`delete old-jobs handler tests should delete all jobs using defaults in parallel 1`] = `"Successfully deleted 2 jobs"`;
+
+exports[`delete old-jobs handler tests should delete all jobs using defaults in parallel 2`] = `
+Object {
+  "fields": Array [
+    "jobname",
+    "jobid",
+    "status",
+  ],
+  "format": "table",
+  "output": Array [
+    Object {
+      "class": "A",
+      "files-url": "www.nowhere.com/restjobs/jobs/files",
+      "job-correlator": "123545asdfadf",
+      "jobid": "TSUxxx",
+      "jobname": "IBMUSER$",
+      "owner": "IBMUSER",
+      "phase": 88,
+      "phase-name": "testagain",
+      "retcode": "CC 0000",
+      "status": "OUTPUT",
+      "subsystem": "JES2",
+      "type": "job",
+      "url": "www.nowhere.com/restjobs/jobs",
+    },
+    Object {
+      "class": "A",
+      "files-url": "www.nowhere.com/restjobs/jobs/files",
+      "job-correlator": "123545asdfadf",
+      "jobid": "JOBxxx",
+      "jobname": "CAUSER$",
+      "owner": "CAUSER",
+      "phase": 88,
+      "phase-name": "testagain",
+      "retcode": "CC 0000",
+      "status": "OUTPUT",
+      "subsystem": "JES2",
+      "type": "job",
+      "url": "www.nowhere.com/restjobs/jobs",
+    },
+  ],
+}
+`;
+
+exports[`delete old-jobs handler tests should delete all jobs using defaults in parallel 3`] = `"Successfully deleted 2 jobs"`;
+
+exports[`delete old-jobs handler tests should delete all jobs using defaults in parallel 4`] = `
+Array [
+  Object {
+    "class": "A",
+    "files-url": "www.nowhere.com/restjobs/jobs/files",
+    "job-correlator": "123545asdfadf",
+    "jobid": "TSUxxx",
+    "jobname": "IBMUSER$",
+    "owner": "IBMUSER",
+    "phase": 88,
+    "phase-name": "testagain",
+    "retcode": "CC 0000",
+    "status": "OUTPUT",
+    "subsystem": "JES2",
+    "type": "job",
+    "url": "www.nowhere.com/restjobs/jobs",
+  },
+  Object {
+    "class": "A",
+    "files-url": "www.nowhere.com/restjobs/jobs/files",
+    "job-correlator": "123545asdfadf",
+    "jobid": "JOBxxx",
+    "jobname": "CAUSER$",
+    "owner": "CAUSER",
+    "phase": 88,
+    "phase-name": "testagain",
+    "retcode": "CC 0000",
+    "status": "OUTPUT",
+    "subsystem": "JES2",
+    "type": "job",
+    "url": "www.nowhere.com/restjobs/jobs",
+  },
+]
+`;
+
+exports[`delete old-jobs handler tests should delete all jobs using defaults sequentially 1`] = `"Successfully deleted 2 jobs"`;
```

Comment:
It would be nice to move away from using snapshots for things like this.
But that will be something we need to do across the board (if we think it's a good idea) 😋